src/amp/lib/permutive.ts

```diff
@@ -0,0 +1,39 @@
+export interface PermutivePayload {
+    'properties.content.Premium'?: string;
+    'properties.content.id'?: string;
+    'properties.content.title'?: string;
+    'properties.content.section'?: string;
+    'properties.content.authors!'?: string;
+    'properties.content.keywords!'?: string;
+    'properties.content.publishedAt'?: string;
+    'properties.user.edition'?: string;
+}
+
+export const generatePermutivePayload = (
+    rawConfig: CommercialConfigType,
+): PermutivePayload => {
+    const publishedAt =
+        rawConfig.webPublicationDate &&
+        typeof rawConfig.webPublicationDate === 'number'
+            ? new Date(rawConfig.webPublicationDate).toISOString()
+            : void 0;
+    const config: { [key: string]: any } = {
+        'properties.content.Premium': rawConfig.isPaidContent,
```

Comment:
This isn't right! Paid content and premium content aren't the same thing. At the moment we don't have any premium content so this will always be false, although it has existed in the past (typically just in the apps).

Reply:
The schema spreadsheet maps premium to paidContent 🤔 Also that means we need to fix `frontend` as it uses `paidContent` as well. On a quick look I can't even find `premium` in our config store, do you know where is it?